core/jazz_create-serverless-service/index.js

```diff
@@ -231,6 +231,11 @@ var getServiceData = (service_creation_data, authToken, configData) => {
             serviceMetadataObj.require_internal_access = service_creation_data.require_internal_access;
         }
 
+        // Store the runtime onto service catalog 
+        if (service_creation_data.runtime.startsWith("python")){
+            serviceMetadataObj.providerRuntime = service_creation_data.runtime;
```

Comment:
I think a better to do this is store `providerRuntime` in `serviceMetadataObj` irrespective of runtime so that it always exist in service metadata (remove the check  - `service_creation_data.runtime.startsWith("python")`).

Reply:
updated